src/Assets/Lists/members.js

```diff
@@ -108,25 +108,103 @@ const memberData = {
         },
         "External Relations": {
             membersList: [
-                
+                {
+                    fullName: "Archana Rajendran",
+                    position: "Director",
+                    emailLink: "a.rajendran@mail.utoronto.ca",
+                },
+                {
+                    fullName: "Chu Qiao",
+                    position: "Director",
+                    emailLink: "chu.qiao@mail.utoronto.ca",
+                    LinkedInLink: "https://www.linkedin.com/in/jocelynqiaochu/",
+                },
+                {
+                    fullName: "Dooyeon Yeom",
+                    position: "Senior Associate",
+                    emailLink: "dooyeon.yeom@mail.utoronto.ca",
+                    LinkedInLink: "https://www.linkedin.com/in/dooyeon-yeom/",
+                },
+                {
+                    fullName: "Manha Siddiqua",
+                    position: "Associate",
+                    emailLink: "manha.siddiqua@mail.utoronto.ca",
+                    LinkedInLink: "https://www.linkedin.com/in/manha-siddiqua-14465b231",
+                },
+                {
+                    fullName: "Royce Luo",
+                    position: "Associate",
+                    emailLink: "roycejb.luo@mail.utoronto.ca",
+                    LinkedInLink: "https://www.linkedin.com/in/royce-luo-63b801244/",
+                },
+                {
+                    fullName: "Tanvi Bansal",
+                    position: "Associate",
+                    emailLink: "tanvi.bansal@mail.utoronto.ca",
+                    LinkedInLink: "https://in.linkedin.com/in/tanvi--bansal",
+                },
+                {
+                    fullName: "Victoria Chen",
```

Comment:
Is she in the team??? How come I don't see her anywhere whether it is in org chart or anything? Let me confirm

Reply:
OK confirmed she is in the team. Also, there is a new ER member that needs to be added, her name is Melissa Cao and you can use her Slack Profile Picture, ignore linkedIn and email for now.